r/Dockerfile

```diff
@@ -26,6 +26,11 @@ RUN R -e "renv::init(bare = TRUE); options(renv.consent = TRUE); renv::restore(l
 COPY renv.lock .
 RUN R -e "options(renv.consent = TRUE); renv::restore(lockfile='renv.lock')"
 
+# install miniconda and python umap-learn for RunUMAP
+ENV RETICULATE_MINICONDA_PATH "/src/renv/python/r-reticulate"
+RUN R -e "reticulate::install_miniconda()"
+RUN R -e "reticulate::conda_install('/src/renv/python/r-reticulate', packages='umap-learn')"
```

Comment:
You should be able to use either `Sys.getenv` or string interpolation here, right?

Reply:
Yes you _should_ be able to use `Sys.getenv('RETICULATE_MINICONDA_PATH')`

Reply:
Yes, not that it's crucial but it would be neater to only have the path once imo